db/migrate/20160331111338_add_created_on_to_container_image.rb

```diff
@@ -0,0 +1,5 @@
+class AddCreatedOnToContainerImage < ActiveRecord::Migration[5.0]
+  def change
+    add_column :container_images, :created_on, :datetime
```

Comment:
created_on is a Rails special column, so I don't think we should use this name for these purposes, otherwise it will just cause confusion.